drivers/linstorvolumemanager.py

```diff
@@ -1741,6 +1747,21 @@ def get_resources_info(self):
 
         return resources
 
+    def get_resource_info(self, volume_uuid: str) -> Dict:
```

Comment:
```suggestion
    def get_resource_info(self, volume_uuid: str) -> Dict[str, Any]:
```

Not the best suggestion but meh. :p I think we should try to be explicit even when it's not simple. Of course, within the limits of readability!